resources/views/projects/tasks/tab.blade.php

```diff
@@ -0,0 +1,285 @@
+		<div class="right aligned sixteen wide column">
+			<button type="button" class="ui button primary" onclick="showTaskModal('new')">New task</button>
+		</div>
+		@include('projects.tasks.form', ['name' => 'new'])
+		@include('projects.tasks.form', ['name' => 'edit'])
+	@endif
+	<div class="sixteen wide column">
+		@include('projects.tasks.index')
+	</div>
+</div>
+@include('projects.tasks.details')
+
+@push('js')
+<script>
+	@if($project->isStakeholder(Auth::id()))
+		function fillTaskDetailsModal(task) {
+			$("#task-edit-btn").data("taskid", task.id);
+			$("#task-details #task-title").text(task.title);
+
+			// status
+			switch(task.task_status) {
+				case 1: // todo
+					$("#task-details #task-status p").text("To-do");
+					$("#task-details #task-status i").removeClass().addClass("small circle green icon");
+					break;
+				case 2: // in-progress
+					$("#task-details #task-status p").text("In progress");
+					$("#task-details #task-status i").removeClass().addClass("small circle yellow icon");
+					break;
+				case 3: // closed
+					$("#task-details #task-status p").text("Closed");
+					$("#task-details #task-status i").removeClass().addClass("small circle blue icon");
+					break;
+			}
+			$("#task-details #task-status p").data("taskstatus", task.task_status);
+
+			// description
+			$("#task-details #task-description").text(task.description);
+
+			// due date
+			const deadline = task.deadline;
+			const currentDatetimeUTC = Date.now();
+			const deadlineDateUTC = new Date(deadline + "Z");
+			const isOverdue = deadlineDateUTC < currentDatetimeUTC;
+			$("#task-details #task-due-date").text(task.deadline).data("datetimeutc", task.deadline);
+			// style due date
+			if(isOverdue) {
+				$("#task-details #task-due-date").addClass("overdue");
+			}
+			else {
+				$("#task-details #task-due-date").addClass("overdue");
+			}
+
+			// task opened details
+			$("#task-details #task-opened-date").text(task.created_at).data("datetimeutc", task.created_at);
+			$("#task-details #task-opened-by").text(task.creator.name);
+
+			// task closed details
+			if(task.task_status == 3) { // task is closed
+				$("#task-details #task-closed-date").text(task.completed_date).data("datetimeutc", task.completed_date);
+				$("#task-details #task-closed-by").text(task.closed_by.name);
+				$("#task-details #task-closed-details").show();
+			}
+			else {
+				$("#task-details #task-closed-details").hide();
+			}
+
+			// format all datetimes to local
+			utcToLocal();
+		}
+
+		function showTaskDetailsModal(taskId) {
+			$.ajax({
+				type: "GET",
+				url: `/tasks/${taskId}`,
+				headers: {
+					'X-CSRF-TOKEN': $('meta[name="csrf-token"]').attr('content')
+				},
+				dataType: 'json',
+				success: function (task) {
+					fillTaskDetailsModal(task);
+					$("#task-details-modal").modal("show");
+				}
+			});
+		}
+
+		$(".task-list").click((e) => {
+			if(!$(e.target).hasClass("task-title")) return;
```

Comment:
Si solo lo quieres ejecutar cuando le pique al `task-title` por que no bindeas el event handler en ese elemento?

Reply:
Porque de esta forma, si agrego elementos dinámicamente (e.g. new tasks) el event listener funciona también para estos nuevos elementos que no estaban cargados cuando se cargó la vista. 

See: [Event delegation in JS](https://medium.com/@bretdoucette/part-4-what-is-event-delegation-in-javascript-f5c8c0de2983)